### PR TITLE
Add multiple keyword feature

### PR DESCRIPTION
**Pull request by keltroth**
I've added a little feature to be able to put multiple keywords, comma separated.

### CODE REVIEW

File: randomwallpaper@iflow.space/settings.ui [hunk 1]
```diff
@@ -985,7 +985,7 @@ You can also define a prefix that will be added to the image URL.</property>
         <property name="visible">1</property>
         <property name="can-focus">1</property>
         <property name="margin-bottom">10</property>
-        <property name="placeholder-text" translatable="yes">Enter a keyword ...</property>
+        <property name="placeholder-text" translatable="yes">Some keywords, comma separated ...</property>
```
Comment (ifl0w) [inline randomwallpaper@iflow.space/settings.ui:988]:
```suggestion
        <property name="placeholder-text" translatable="yes">Enter keywords... (comma separated)</property>
```

File: randomwallpaper@iflow.space/settings.ui [hunk 2]
```diff
@@ -835,7 +835,7 @@ You can also define a prefix that will be added to the image URL.</property>
         <property name="visible">1</property>
         <property name="can-focus">1</property>
         <property name="margin-bottom">10</property>
-        <property name="placeholder-text" translatable="yes">Some keyword ...</property>
+        <property name="placeholder-text" translatable="yes">Some keywords, comma separated ...</property>
```
Comment (ifl0w) [inline randomwallpaper@iflow.space/settings.ui:838]:
```suggestion
        <property name="placeholder-text" translatable="yes">Enter keywords... (comma separated)</property>
```

**Review by ifl0w — CHANGES_REQUESTED** [review]
Good improvement, thanks! :)

You might want to improve the wording in the placeholder text. 
My suggestion is only one option. I'll not stand in the way if you find better words for it.

**Comment by keltroth** [comment]
@ifl0w I will not stand neither :) maybe we should use the same wording with subreddits... 
`<property name="label" translatable="yes">Subreddits (separate with a comma)</property>`

**Review by ifl0w — APPROVED** [review]
You got a point there. Optimally, it should be consistent with the subreddit placeholder ... ^^

I'll update the subreddit placeholder to "Subreddits ... (comma separated)" and remove the "Enter" at the beginning of the other placeholders in a separate commit. :) 

Thanks for the contribution!

**Pull request merged**